Explorer/Assets/DCL/Communities/CommunitiesCard/CommunityCardController.cs

```diff
@@ -115,12 +123,42 @@ public override void Dispose()
             eventListController?.Dispose();
         }
 
+        private void OnDeleteCommunityRequested()
+        {
+            communityOperationsCancellationTokenSource = communityOperationsCancellationTokenSource.SafeRestart();
+            DeleteCommunityAsync(communityOperationsCancellationTokenSource.Token).Forget();
+            return;
+
+            async UniTaskVoid DeleteCommunityAsync(CancellationToken ct)
```

Comment:
Can't do this. Your options:
- Change return type to UniTask
- Wrap the entire contents of the method in try-catch and catch everything except OperationCanceledException.